packages/ui/src/components/Button/button.tsx

```diff
@@ -13,18 +13,20 @@ type CondoButtonProps = {
     type: 'primary' | 'secondary',
     children?: string
     stateless?: boolean
+    textAlign?: 'left' | 'center' | 'right'
 }
 
 export type ButtonProps = Omit<DefaultButtonProps, 'shape' | 'size' | 'style' | 'ghost' | 'type' | 'prefix' | 'prefixCls'>
 & CondoButtonProps
 
 const Button = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
-    const { type, className, icon, children, onClick, stateless, ...rest } = props
+    const { type, className, icon, children, onClick, stateless, textAlign = 'center', ...rest } = props
```

Comment:
It's a bit strange, can you explain why? Screenshots are welcome

Reply:
fixed